predictor_robojudge.ipynb

```diff
@@ -40,24 +41,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def load_dataset(url):\n",
```

Comment:
`load_dataset` is now a function in `predictor_scoring`, and it takes an additional argument to specify the file containing the list of countries and regions to keep.